ecs-cli/modules/commands/attributechecker/attribute_checker_command.go

```diff
@@ -0,0 +1,31 @@
+package attributecheckercommand
+
+import (
+	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/cli/attributechecker"
+	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands/flags"
+	"github.com/urfave/cli"
+)
+
+// AttributecheckerCommand checks if all Capabilities/attributes are available to run the task on a specified Cluster or on a given Container Instance specified.
+func AttributecheckerCommand() cli.Command {
+	return cli.Command{
+		Name:         "check-attributes",
+		Usage:        "Checks if a given task def attributes are available to RunTask on a specified Cluster or on a given Container Instance and outputs missing attribute(s).",
```

Comment:
Let's change this to:

"Checks if a given list of container instances can run a given task definition by checking their attributes. Outputs attributes that are required by the task definition but not present on the container instances."